modules/hrm/views/js-templates/employment-type.php

```diff
@@ -0,0 +1,53 @@
+<?php
+$id = isset( $_GET['id'] ) ? absint( wp_unslash( $_GET['id'] ) ) : 0;
+$value = '';
+
+if ( $id ) {
+    $employee = new \WeDevs\ERP\HRM\Employee( $id );
+    $value = $employee->get_type();
+}
+?>
+
+<div class="type-form-wrap">
+    <div class="row">
+        <?php
+        erp_html_form_input( [
+            'label'    => __( 'Date', 'erp' ),
+            'name'     => 'date',
+            'value'    => gmdate( 'Y-m-d', current_time( 'timestamp' ) ),
```

Comment:
a more modern way to get date is erp_current_datetime()->format('Y-m-d'), also use of current_time('timestamp') is discouraged.